app/src/scenes/SubCategories/scenes/Items/interfaces.ts

```diff
@@ -11,7 +11,9 @@ export interface ItemStateProps {
 export interface Item {
   id: number;
   link: string;
-  translations: ItemTranslation[];
+  name: string;
+  language: string;
+  description: string;
 }
```

Comment:
We still have translations array on the payload. The only difference is now we have two specific columns for the English language to be used as the default and mandatory. 
We need to change these interfaces in align with that payload. For example, item interface should be changed into this,
```
export interface Item {
  id: number;
  link: string;
  name: string;
  description: string;
  translations: ItemTranslation[];
}
```
Apply necessary changes to the other interfaces in the same manner.
Got it?

Reply:
Done! Thanks @Gravewalker666